### PR TITLE
Add new question to AAA FAQ

### PR DESCRIPTION
**Pull request by machacekondra**
This PR add new question for AAA FAQ:

```
Is it possible to use modify search filter for users?
```

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina @JohnMarksRH

File: source/develop/release-management/features/infra/aaa_faq.html.md [hunk 1]
```diff
@@ -115,3 +115,18 @@ Yes, it's possible with *apache mod_auth_cas* module. Full blog post is availabl
 
 Unfortunatelly it's not supported, yet. You can track [this](https://bugzilla.redhat.com/show_bug.cgi?id=829292) bugzilla for more information.
 
+## Is it possible to use modify search filter for users?
+Yes it's possible with a little hack to you properties file.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: Suggest:
"Yes it's possible, with a little hack to your properties file."

File: source/develop/release-management/features/infra/aaa_faq.html.md [hunk 2]
```diff
@@ -115,3 +115,18 @@ Yes, it's possible with *apache mod_auth_cas* module. Full blog post is availabl
...
+Please append following lines to your profile configuration:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: Suggest:
"Please append the following lines to your profile configuration:"

File: source/develop/release-management/features/infra/aaa_faq.html.md [hunk 3]
```diff
@@ -115,3 +115,18 @@ Yes, it's possible with *apache mod_auth_cas* module. Full blog post is availabl
...
+```
+sequence.simple-query-principals.005.description = modify filter to search only by uid
+sequence.simple-query-principals.005.type = regex
+sequence.simple-query-principals.005.regex.value = ${seq:filter}
+sequence.simple-query-principals.005.regex.flags = a
+sequence.simple-query-principals.005.regex.pattern = \\(givenName=[^)]*\\)|\\(sn=[^)]*\\)|\\(displayName=[^)]*\\)
+sequence.simple-query-principals.005.regex.replacement.filter = 
+```
+
+This example will replaces occurances of givenName, sn and displayName and it's values by empty string.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: Suggest:
"This example will replace occurrences of givenName, sn, and displayName and its values with an empty string."

File: source/develop/release-management/features/infra/aaa_faq.html.md [hunk 4]
```diff
@@ -115,3 +115,18 @@ Yes, it's possible with *apache mod_auth_cas* module. Full blog post is availabl
...
+So only uid persist in filter. You can modify it as you want just by changing the regex expression.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: Suggest:
"Only uid will persist in the filter. You can modify it as you like, simply by changing the regex expression.

**Comment by machacekondra** [comment]
updated

File: source/develop/release-management/features/infra/aaa_faq.html.md [hunk 5]
```diff
@@ -115,3 +115,18 @@ Yes, it's possible with *apache mod_auth_cas* module. Full blog post is availabl
 
 Unfortunatelly it's not supported, yet. You can track [this](https://bugzilla.redhat.com/show_bug.cgi?id=829292) bugzilla for more information.
 
+## Is it possible to use modify search filter for users?
```
Comment (mwperina) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: Shouldn't it be: "Is it possible to modify users search filter"?

Comment (machacekondra) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: done

**Comment by JohnMarksRH** [comment]
@mwperina @machacekondra I am not familiar with this features, so my comments are only for grammar.
Please check that this is what you mean:
"Is it possible to use modify search filter for users?"
Maybe you just want to say:
"Is it possible to modify the search filter for users"?

**Comment by mwperina** [comment]
> @mwperina @machacekondra I am not familiar with this features, so my comments are only for 
> grammar.
> Please check that this is what you mean:
> "Is it possible to use modify search filter for users?"
> Maybe you just want to say:
> "Is it possible to modify the search filter for users"?

John, this was already addressed by Ondra, please recheck. If OK, please merge as it's seems OK to me.

Thanks

Martin

@JohnMarksRH

**Comment by mwperina** [comment]
@JohnMarksRH Could you please merge?

**Pull request merged**